tools/flagd-http-connector/CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
```

Comment:
I think we can clear this changelog except for the title?

Reply:
done